src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,23 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+}
```

Comment:
May be better if you move this to a normalize file as it does not reset but adds new features.

Reply:
Agree with Anastasia